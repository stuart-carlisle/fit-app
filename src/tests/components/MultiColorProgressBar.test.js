import { shallow } from 'enzyme'
import React from 'react'
import MultiColorProgressBar from '../../components/MultiColorProgressBar'
 
test('Should render the multicolor progress bar correctly',()=>{
    const wrapper = shallow(<MultiColorProgressBar />)
    expect(wrapper).toMatchSnapshot()
})