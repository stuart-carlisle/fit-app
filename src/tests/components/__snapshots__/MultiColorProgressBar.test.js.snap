// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the multicolor progress bar correctly 1`] = `
<div
  className="multicolor-bar"
>
  <div
    className="bars"
  />
</div>
`;
